src/backend/dx11/src/device.rs

```diff
@@ -992,7 +1081,14 @@ impl hal::Device<Backend> for Device {
                     Usage: usage,
                     BindFlags: bind,
                     CPUAccessFlags: cpu,
-                    MiscFlags: 0
+                    // unfortunately no better way of doing this, since creating a cube view
```

Comment:
actually, we do know that from `image.flags`

Reply:
Must've missed it!